requirements-agent-release.txt

```diff
@@ -17,6 +17,7 @@ datadog-calico==1.0.1
 datadog-cassandra-nodetool==1.11.0
 datadog-cassandra==1.15.1
 datadog-ceph==2.7.0; sys_platform != 'win32'
+datadog-cert-manager==2.2.0; sys_platform != 'darwin'
```

Comment:
We can update this even if it's not released https://github.com/DataDog/integrations-core/pull/11789#issuecomment-1092136853